src/python/platforms/fuchsia/device.py

```diff
@@ -74,25 +82,24 @@ def qemu_setup():
       '-monitor', 'none',
       '-append', '"kernel.serial=legacy TERM=dumb"',
       '-machine', 'q35',
-      '-enable-kvm',
       '-display', 'none',
-      '-cpu', 'host,migratable=no',
+      '-cpu', 'Haswell,+smap,-check,-fsgsbase',
```

Comment:
Could you explain what this cpu option is doing? Is this what's turning off KVM if it's an unsupported CPU ?